circuits/cpp/src/aztec3/circuits/abis/tx_request.hpp

```diff
@@ -14,11 +14,13 @@ using aztec3::utils::types::CircuitTypes;
 using aztec3::utils::types::NativeTypes;
 
 template <typename NCT> struct TxRequest {
+    typedef typename NCT::secp256k1_point secp256k1_point;
     typedef typename NCT::address address;
     typedef typename NCT::fr fr;
     typedef typename NCT::boolean boolean;
 
     address from = 0;
+    secp256k1_point from_public_key;
```

Comment:
As discussed, let's move this to the signed tx request struct

Reply:
Okay so I did try moving this to `signed_tx_request`. It leads to a lot of changes on the TS side and I think its better to do that as its own PR).